apps/web-giddh/src/assets/css/style-2.scss

```
@@ -5735,26 +5841,26 @@ span.red-text {
     }
 }
 .ratio-chart-block:first-child span.pie-text_center {
-    color: $primary;
+    color: var(--color-blue);
 }
 .ratio-chart-block:nth-child(2) span.pie-text_center {
-    color: $vivid-orange;
+    color: var(--color-ratio-chart-block-nth-child2-span-pie-text_center);
 }
 .ratio-chart-block:nth-child(3) span.pie-text_center {
-    color: $violet-red;
+    color: var(--color-ratio-chart-block-nth-child3-span-pie-text_center);
```

Avoid `snake_case` in class names
```suggestion
    color: var(--color-ratio-chart-block-nth-child3-span-pie-text-center);
```